app/src/main/java/org/p2p/wallet/history/ui/history/adapter/HistoryAdapterPagingController.kt

```diff
@@ -0,0 +1,35 @@
+package org.p2p.wallet.history.ui.history.adapter
+
+import org.p2p.wallet.common.ui.recycler.PagingState
+
+class HistoryAdapterPagingController(
```

Comment:
Are you sure this worth it? 
Maybe it could be useful functions in these be called in all paging adapters but if it's only within HistoryAdapter, then this logic might be more helpful to understand the adapter's logic

Reply:
Yea, im sure
smth should control and hold logic of PagicState of this adapter.
adapter has enough responsibility and logic inside already (creating, binding, adding different type of views), so it's okay to decompose it